modules/govuk/files/etc/govuk/import_mapit_data.sh

```diff
@@ -1,27 +1,38 @@
 #!/bin/bash
 
+set -ex
+
 function stop_mapit_services () {
   # Stop Mapit Services
-  sudo service nginx stop
+  sudo service nginx stop || true
   # Stop mapit and collectd which are using the Mapit database so that we can drop it
-  sudo service mapit stop
-  sudo service collectd stop
+  sudo service mapit stop || true
+  sudo service collectd stop || true
   # # Make sure that cached mapit responses are cleared when the database is updated
-  sudo service memcached restart
+  sudo service memcached restart || true
 }
 
 function restart_mapit_services () {
   # Restart Mapit Services
-  sudo service collectd start
-  sudo service mapit start
-  sudo service nginx start
+  sudo service collectd start || true
+  sudo service mapit start || true
+  sudo service nginx start || true
 }
 
-stop_mapit_services
-# This script needs to run as a user that can sudo to become the postgres
-# user.  It's easiest if this is passwordless, so we can't use the deploy user
-# who can't sudo without a password
-# Run 'govuk_setenv mapit ./import-db-from-s3.sh' from the '/var/apps/mapit' as
 
-cd /var/apps/mapit && govuk_setenv mapit ./import-db-from-s3.sh
+function import_db_from_s3 () {
+  # get the data from s3
+  aws s3 cp --no-sign-request s3://govuk-custom-formats-mapit-storage-production/source-data/2021-05/mapit-may-2021-update.sql.gz /tmp/mapit.sql.gz
```

Comment:
Btw the use of a fixed location in /tmp isn't good practice from a security standpoint, but in this case I don't think it's a big deal because if a bad actor is on the box they're not gonna need any help from a tempfile TOCTOU in this script 😅 Just mentioning for future reference - it might matter in some other situation you encounter.